net-im/signal-desktop-bin/Manifest

```diff
@@ -1 +1,2 @@
 DIST signal-desktop_1.39.2_amd64.deb 109799098 BLAKE2B 83d1b047141ae94a1b246792bf92f0e7baa92b938df9dfbf790e90e161e9cb4f9176544deceb4507a65181113c00e6cb6d367030079f9137ffcb0f177e14c95b SHA512 21208910bc48f4def5fad6262fa878b8c6f1ae7f2a2ffa3167c2992e0dc220ba67da43d23adb0b3ad1aa8660927065e97e6bcf7b289fa19010f5fc9e94c6d56e
+DIST signal-desktop_1.39.4_amd64.deb 109787466 BLAKE2B c64efe741c93017be3ad619bac4182a10231c152f76653513d9a749f965c8990d6b2736bcb48ff8db6f30c8756ee1ef6f616c464d129299489cb1246c6ce955e SHA512 c8ee94f8c706e7f5c57ace75101d809d01753d7517197ec98d82106f8279c0099b502251c378643fa6b6d543bcb494cd011128caefa12ab69ad0feaca3ce7732
```

Comment:
```
>>> Emerging (1 of 1) net-im/signal-desktop-bin-1.39.4::testworld
>>> Downloading 'https://updates.signal.org/desktop/apt/pool/main/s/signal-desktop/signal-desktop_1.39.4_amd64.deb'
--2021-01-15 12:04:06--  https://updates.signal.org/desktop/apt/pool/main/s/signal-desktop/signal-desktop_1.39.4_amd64.deb
Resolving updates.signal.org... 13.33.243.23, 13.33.243.93, 13.33.243.39, ...
Connecting to updates.signal.org|13.33.243.23|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-01-15 12:04:06 ERROR 404: Not Found.

!!! Couldn't download 'signal-desktop_1.39.4_amd64.deb'. Aborting.
 * Fetch failed for 'net-im/signal-desktop-bin-1.39.4', Log file:
 *  '/var/tmp/portage/portage/net-im/signal-desktop-bin-1.39.4/temp/build.log'

```
It doesn't look like new version has been released either. What's up with this?

Reply:
Signal is already at version 1.39.5 (this PR is nearly a month old). They remove their old versions quite quickly from their servers. I'll update it to the new version.

Reply:
I updated this to target 1.39.5.